### ISSUE TITLE
一些问题

### ISSUE THREAD

**Issue by LiHaoyu549** [JianghaiSCU/LightenDiffusion#12]
1. 关于第一阶段训练：请问是直接对decom.py训练吗？因为看到论文训练第一阶段是说“一对低光作为输入，并且冻结diffusion的参数”，第一阶段训练编解码器和Retinex分解应该跟diffusion无关吧。是不是只用训练decom.py，即CTDN输出的R和L相乘，输入给解码端，并不需要输入diffusion？也可能我有误解，需要作者您解答一下，谢谢！
2. 关于编解码器，我看到代码对encoder三个尺度的低光特征运用跳转连接到decoder端，这不会引入低光影响解码吗？
3. 在restoration.py文件中27-34行代码如下，x_cond是低光图，`pred_x = self.diffusion.model(torch.cat((x_cond, x_cond), dim=1))["pred_x"][:, :, :h, :w]`，不是应该cat低光和正常光吗，怎么cat两个一样的？
```
x_cond = x[:, :3, :, :].to(self.diffusion.device)
                b, c, h, w = x_cond.shape
                img_h_64 = int(64 * np.ceil(h / 64.0))
                img_w_64 = int(64 * np.ceil(w / 64.0))
                x_cond = F.pad(x_cond, (0, img_w_64 - w, 0, img_h_64 - h), 'reflect')

                t1 = time.time()
                pred_x = self.diffusion.model(torch.cat((x_cond, x_cond),
                                                        dim=1))["pred_x"][:, :, :h, :w]
```

**Comment by JianghaiSCU** [comment]
1.是的，一阶段不需要训练diffusion；
2.不会影响，只是利用low-light feature帮组decoder解码；
3. infer只需要low-light参与

**Comment by LiHaoyu549** [comment]
谢谢！

**Comment by d-linlin** [comment]
> 1. 关于第一阶段训练：请问是直接对decom.py训练吗？因为看到论文训练第一阶段是说“一对低光作为输入，并且冻结diffusion的参数”，第一阶段训练编解码器和Retinex分解应该跟diffusion无关吧。是不是只用训练decom.py，即CTDN输出的R和L相乘，输入给解码端，并不需要输入diffusion？也可能我有误解，需要作者您解答一下，谢谢！
> 2. 关于编解码器，我看到代码对encoder三个尺度的低光特征运用跳转连接到decoder端，这不会引入低光影响解码吗？
> 3. 在restoration.py文件中27-34行代码如下，x_cond是低光图，`pred_x = self.diffusion.model(torch.cat((x_cond, x_cond), dim=1))["pred_x"][:, :, :h, :w]`，不是应该cat低光和正常光吗，怎么cat两个一样的？
> 
> ```
> x_cond = x[:, :3, :, :].to(self.diffusion.device)
>                 b, c, h, w = x_cond.shape
>                 img_h_64 = int(64 * np.ceil(h / 64.0))
>                 img_w_64 = int(64 * np.ceil(w / 64.0))
>                 x_cond = F.pad(x_cond, (0, img_w_64 - w, 0, img_h_64 - h), 'reflect')
> 
>                 t1 = time.time()
>                 pred_x = self.diffusion.model(torch.cat((x_cond, x_cond),
>                                                         dim=1))["pred_x"][:, :, :h, :w]
> ```

你好，请问你第一阶段训练成功了吗，可否提供一份训练代码？谢谢！
